hpat/distributed_api.py

```diff
@@ -330,30 +330,36 @@ def const_slice_getitem(arr, slice_index, start, count):
 
 @overload(const_slice_getitem)
 def const_slice_getitem_overload(arr, slice_index, start, count):
+    # slice.index start/stop contain absolute indexes of the slice in input array
+    # start/count define part of the array processed by this processor
+
+    # TODO: should this also handle slices not staring from zero?
```

Comment:
Could you convert his into a proper doc-string?
What is this functions supposed to do? ...